charts/trusted-artifact-signer/templates/tests/test-sign-verify.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: trusted-artifact-signer-test-sign-verify
+  labels:
+    app.kubernetes.io/component: trusted-artifact-signer
+  annotations:
+      helm.sh/hook: test
+spec:
+    initContainers:
+      - name: buildah
+        image: quay.io/buildah/stable
+        command: ["/bin/sh", "-c"]
+        args:
+        - |
+            buildah pull alpine:latest
+            buildah tag alpine:latest ttl.sh/sigstore-test:5m
+            buildah push ttl.sh/sigstore-test:5m
+        securityContext:
+            privileged: true
+            runAsUser: 1000
+    containers:
+      - name: cosign
+        image: "{{ .Values.configs.cosign.image.registry }}/{{ .Values.configs.cosign.image.repository }}:{{ .Values.configs.cosign.image.version }}"
+        env:
+        - name: OPENSHIFT_APPS_SUBDOMAIN
+          value: {{ .Values.configs.cosign.appsSubdomain }}
+        - name: OIDC_AUTHENTICATION_REALM
+          value: "sigstore"
+        - name: FULCIO_URL
+          value: "https://fulcio.$(OPENSHIFT_APPS_SUBDOMAIN)"
+        - name: OIDC_ISSUER_URL
+          value: "https://keycloak-keycloak-system.$(OPENSHIFT_APPS_SUBDOMAIN)/auth/realms/$(OIDC_AUTHENTICATION_REALM)"
+        - name: REKOR_URL
+          value: "https://rekor.$(OPENSHIFT_APPS_SUBDOMAIN)"
+        - name: TUF_URL
+          value: "https://tuf.$(OPENSHIFT_APPS_SUBDOMAIN)"
+        command: ["/bin/sh", "-c"]
+        args:
+          - |
+            cosign initialize --mirror=$TUF_URL --root=$TUF_URL/root.json
+            TOKEN=$(curl -X POST -H "Content-Type: application/x-www-form-urlencoded" -d "username=jdoe" -d "password=secure" -d "grant_type=password" -d "scope=openid" -d "client_id=sigstore" $OIDC_ISSUER_URL/protocol/openid-connect/token |  sed -E 's/.*"access_token":"([^"]*).*/\1/')
```

Comment:
Is this an issue to use hardcoded credentials like this? Im sure our routes / endpoints are available throughout the `securesign` repos in agregate, and couldn't someone sign code using this user against our stack as someone else?

Reply:
I'm not sure I understand your point exactly.
I followed the installation path described at https://github.com/securesign/sigstore-ocp/blob/main/quick-start-with-keycloak.md
As you can see there is a step to install keycloak resources from https://github.com/securesign/sigstore-ocp/tree/main/keycloak/resources. If you do, you end up with the user I used to sign the container in the test (see https://github.com/securesign/sigstore-ocp/blob/main/keycloak/resources/user.yaml#L14)

I know a pre-configured user can be seen as a pitfall, but I believe the keycloack scenario is worth covering.